tests/cupy_tests/core_tests/fusion_tests/test_array.py

```diff
@@ -0,0 +1,297 @@
+import unittest
+
+import numpy
+import six
+
+from cupy import testing
+from cupy_tests.core_tests.fusion_tests import fusion_utils
+
+
+class FusionArrayTestBase(unittest.TestCase):
+
+    def _get_argument(self, xp, dtype, seed, value_type):
+        dtype = numpy.dtype(dtype)
+        if value_type == 'array':
+            x = testing.shaped_random((3, 4), xp, dtype, scale=5, seed=seed)
+            # Avoid zero-division
+            x[x == 0] = 1
```

Comment:
How about doing this only when necessary (in division tests), or adding TODO for that?